test/prism_regression/lambda.rb

```diff
@@ -0,0 +1,21 @@
+# typed: false
+
+-> { 123 }
```

Comment:
Can you switch up the test data, parameter names, etc. between the various cases?

I use them to find the correspondence between the Ruby syntax and the particular expected tree it corresponds to